app/templates/dashboard/builds.hbs

```diff
@@ -1,5 +1,12 @@
 <ul class="repo-list">
   {{#each (sort-by "startedAt:desc" this.model) as |build|}}
     <MyBuild data-test-my-build @build={{build}} />
+  {{else if (and this.model (not this.model.length))}}
+    <NoBuilds 
+      @title="No builds started yet"
+      @subtitle="Want to start testing on Travis CI?"
+    />
+  {{else}}
+    <LoadingIndicator/>
```

Comment:
I'm not sure about this change, for me it always shows loading indicator when the list is empty. I'd suggest just adding a `-loading` template for loading state. You can read more about the route substates here: https://guides.emberjs.com/v3.11.0/routing/loading-and-error-substates/

Reply:
That's because `repositories` route is handled at dashboard root (notice `path: '/'`)

Reply:
You can add `builds-loading.hbs` and Ember will catch it and handle properly.

Reply:
I guess you're right. The model hook uses `store.filter` which doesn't return a promise, but returns a collection right away. It looks like a general issues that we currently have regarding the loading states. I'm going to investigate and fix it today (well, start at least 😅), so let's keep it assuming that the model returns promise :) And I'll keep in mind to check this case as well while working on my fix. How does it sound to you @emmanuel-ci?